File: yggdrasil/examples/conditional_io/src/conditional_io_modelB2.R
modelB_function2 <- function(in_val) {
  # Only valid if in_val > 2
  out_val <- 2 * in_val^2
  print(sprintf("modelB_function2(%f) = %f", in_val, out_val))
  return(list(in_val, out_val))
}

File: yggdrasil/examples/conditional_io/src/conditional_io_modelB2.m
function [in_val, out_val] = conditional_io_modelB2(in_val)
  % Only valid if in_val > 2
  out_val = 2 * in_val^2;
  disp(sprintf('modelB_function2(%f) = %f', separateUnits(in_val), separateUnits(out_val)));
end
